docs/PacketFence_MSPKI_Quick_Install_Guide.asciidoc

```diff
@@ -0,0 +1,381 @@
+Microsoft PKI (MSPKI) Quick Installation Guide
+==============================================
+:encoding: UTF-8
+:lang: en
+:doctype: book
+
+include::includes/global-attributes.asciidoc[]
+
+About this Guide
+----------------
+
+This guide has been created to give a quick start to configure the Microsoft PKI with PacketFence 5.4+. This guide does not include advanced troubleshooting of EAP-TLS connections. Refer to the relevant documentation of EAP-TLS, RADIUS and OpenSSL for advanced features.
+
+Assumptions
+-----------
+
+* You have at least one server with PacketFence 5.4 or later.
+* The server already has a properly configured switch with 802.1X support.
+* The PacketFence RADIUS server is working in your environment.
+* You have a Microsoft Windows 2008 R2 Entreprise server installed.
+* In this guide you will see a lot of use of <ServerDNSName>, most of the MSPKI services requires in their configuration to use the FQDN of the server and not his IP.
+* The PacketFence management IP will be 192.168.1.5.
+* The RADIUS shared secret is "useStrongerSecret".
+
+Installation
+------------
+
+Step 1: Install Active Directory Certificate Service (ADCS)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+NOTE: This will cover the configuration for Active Directory Certificate Services (ADCS) on Microsoft Windows 2008 R2 Entreprise. This configuration has been tested to be integreted with PacketFence. The installation of ADCS is not cover by this guide see the Microsoft white paper about it for more information (http://social.technet.microsoft.com/wiki/contents/articles/9063.network-device-enrollment-service-ndes-in-active-directory-certificate-services-ad-cs.aspx).
+
+For the integration with PacketFence the following subroles need to be installed in ADCS:
+
+* Certification Authorithy Web Enrollment
+* Network Device Enrollment Service
+* Online Responder
+
+Before start the configuration, a hotfix is necessary due to a Microsft issue. After restarting the service ADCS, the server cannot enroll new certificate with the error message: "The RPC Server is unavailable". The hotfix is available here: https://support.microsoft.com/en-us/kb/2633200
+
+Communication between MSPKI and PacketFence will be using port 80 with this configuration.
+
+Configuring Network Device Enrollement Service (NDES)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+NOTE: In this guide you will see a lot of use of <ServerDNSName>, most of the MSPKI services requires in their configuration to use the FQDN of the server and not his IP.
+
+Our main activity in the deployment of ADCS is to configure Network Enrollement Device Service (NDES) which will allow us to exchange certificate with the MS PKI server via Simple Certificate Exchange Protocol (SCEP).
+
+Every configuration change has to be done by an account with administrative privileges.
+
+To allow the use of NDES as a Pki-Provider in BOYD deployement there is multiple changes.
```

Comment:
You know what, let's just forget this sentence and replace it with.

Allowing the use of NDES as a PKI provider in BYOD deployment requires the following changes